src/test/java/christmas/validation/OrderInputValidatorTest.java

```diff
@@ -0,0 +1,64 @@
+package christmas.validation;
+
+import static org.assertj.core.api.Assertions.assertThatThrownBy;
+
+import christmas.view.validation.OrderInputValidator;
+import org.junit.jupiter.api.DisplayName;
+import org.junit.jupiter.api.Test;
+import org.junit.jupiter.params.ParameterizedTest;
+import org.junit.jupiter.params.provider.ValueSource;
+
+class OrderInputValidatorTest {
+
+    @DisplayName("빈 값을 입력하면 예외가 발생한다.")
+    @ParameterizedTest
+    @ValueSource(strings = {"", " "})
+    void validateEmptyTest(String input) {
+        assertThatThrownBy(() -> OrderInputValidator.validate(input))
+                .isInstanceOf(IllegalArgumentException.class);
```

Comment:
```suggestion
        assertThatIllegalArgumentException().
                isThrownBy(() -> OrderInputValidator.validate(input));
```
assertj의 assertThatIllegalArgumentException 메소드를 사용해도 중복제거와 가독성에 좋을 것 같습니다!

Reply:
오호 이런 메서드도 있군요 커비해갑니다 ㅎㅎ 👍🏻